apps/examples/performance/Makefile

```diff
@@ -51,4 +51,38 @@
 
 MENUDESC = "Performance"
 
-include $(APPDIR)/Directory.mk
+include $(APPDIR)/Make.defs
```

Comment:
Have this Makefile's own set of rues, even if dummy ones, instead of including the APPDIR Directory.mk file for rules